shared/checks/oval/installed_OS_is_rhv4.xml

```diff
@@ -0,0 +1,48 @@
+<def-group>
+  <definition class="inventory"
+  id="installed_OS_is_rhv4" version="1">
+    <metadata>
+      <title>Red Hat Virtualization 4</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <reference ref_id="cpe:/o:redhat:virtualization:4"
+      source="CPE" />
+      <description>The operating system installed on the system is
+      Red Hat Virtualization Host 4.</description>
```

Comment:
rename OVAL to ``installed_OS_is_rhvh4`` ? 

some way to indicate RHV-H vs RHV-M

Reply:
@shawndwells technically `rhvh` isn't accurate because there is also `rhelh`.

Reply:
@shawndwells also I am confirming with rhv guys on how to check if rhevm is installed.

Reply:
@shawndwells I expect this to change in the future but can remain as is since this is all in draft and not finalized.